decidim-accountability/lib/decidim/accountability/result_serializer.rb

```diff
@@ -17,10 +17,7 @@ def initialize(result)
       def serialize
         {
           id: result.id,
-          taxonomies: {
-            id: result.taxonomies.map(&:id),
-            name: result.taxonomies.map(&:name)
-          },
+          taxonomies: serialize_taxonomies(result),
```

Comment:
There is an between resource and result at line 51. Either way, we could remove the parameter from the serialize_taxonomies . Going further, we could rename `serialize_taxonomies` as taxonomies, and remove this method assignment all together.